### ISSUE TITLE
Can't specify custom source checkstyle.xml

### ISSUE THREAD

**Issue by wheelerlaw** [anshulverma/gradle-estilo-plugin#3]
It would be nice to be able to start with something other than the `google` or `sun` one. In essence, if we could just specify the file location instead of using a name.

**Comment by anshulverma** [comment]
@wheelerlaw thats a great idea! Would you be interested in adding this feature?

**Comment by wheelerlaw** [comment]
Yes I would. I will get started on the feature and create a pull request.

>

**Comment by anshulverma** [comment]
Great! Feel free to ping me here if have any questions

**Comment by oyewaleoyelami** [comment]
@wheelerlaw any update on this feature ?

**Comment by wheelerlaw** [comment]
There is a PR on this repo that has some WIP changes associated with it, but I don’t have the bandwidth right now to complete it.
